src/components/Summary/DocumentsDelivery.vue

```diff
@@ -39,21 +27,32 @@
               label="Optional Email"
               hint="Example: name@email.com"
               persistent-hint
+              validate-on-blur
```

Comment:
So does this work as expected -- the validation rules are not evaluated until the user leaves this text field?

Reply:
Yep!